test/test_sigridci_runner.py

```diff
@@ -43,62 +43,47 @@ def testForceLowerCaseForCustomerAndSystemName(self):
         self.options.system = "NOOT"
 
         apiClient = SigridApiClient(self.options)
-        
+
         self.assertEqual(apiClient.urlCustomerName, "aap")
         self.assertEqual(apiClient.urlSystemName, "noot")
-        
-    def testValidateSystemNameAccordingToRules(self):
-        self.assertTrue(SigridCiRunner.isValidSystemName("noot", "aap"))
-        self.assertTrue(SigridCiRunner.isValidSystemName("noot", "aap-noot"))
-        self.assertTrue(SigridCiRunner.isValidSystemName("noot", "aap123"))
-        self.assertTrue(SigridCiRunner.isValidSystemName("noot", "AAP"))
-        self.assertTrue(SigridCiRunner.isValidSystemName("noot", "a" * 59))
-        self.assertTrue(SigridCiRunner.isValidSystemName("noot", "aa"))
-
-        self.assertFalse(SigridCiRunner.isValidSystemName("noot", "aap_noot"))
-        self.assertFalse(SigridCiRunner.isValidSystemName("noot", "a"))
-        self.assertFalse(SigridCiRunner.isValidSystemName("noot", "$$$"))
-        self.assertFalse(SigridCiRunner.isValidSystemName("noot", "-aap"))
-        self.assertFalse(SigridCiRunner.isValidSystemName("noot", "a" * 65))
-        self.assertFalse(SigridCiRunner.isValidSystemName("noot", "aap--aap"))
-        
+
     def testValidateToken(self):
```

Comment:
Move this test case to dedicated test class of SigridApiClient?

Reply:
OK. Most of the actual API interaction is covered by integration tests, but you're right this logic doesn't belong to the runner itself.